build/myget-release.yml

```diff
@@ -11,8 +11,6 @@ variables:
     value: '2.2.105'
   - name: 'Project'
     value: 'Arcus.Security'
-  - name: 'Package.Version.ManualTrigger'
```

Comment:
I think ideally we add the variable as a comment to show that it exists, but is configured on the build itself.

We do that for AD secret for example - https://github.com/arcus-azure/arcus.security/blob/master/build/ci-build.yml#L18

Reply:
Ah, OK; good suggestion!